editor/editor_inspector.cpp

```diff
@@ -88,17 +88,17 @@ Size2 EditorProperty::get_minimum_size() const {
 
 	if (keying) {
 		Ref<Texture2D> key = get_editor_theme_icon(SNAME("Key"));
-		ms.width += key->get_width() + get_theme_constant(SNAME("hseparator"), SNAME("Tree"));
+		ms.width += key->get_width() + get_theme_constant(SNAME("h_separation"), SNAME("Tree"));
```

Comment:
That would be for a future PR, but we should probably move these and other theme constants to a ThemeCache struct like we have in some other editor controls (e.g. `EditorFileDialog`).